internal/bminventory/inventory_test.go

```diff
@@ -2579,46 +2578,16 @@ var _ = Describe("cluster", func() {
 
 			newClusterName := "day1-cluster-new-name"
 
-			reply = bm.UpdateCluster(ctx, installer.UpdateClusterParams{
+			reply = bm.V2UpdateCluster(ctx, installer.V2UpdateClusterParams{
 				ClusterID: *c.ID,
-				ClusterUpdateParams: &models.ClusterUpdateParams{
+				ClusterUpdateParams: &models.V2ClusterUpdateParams{
 					Name:          &newClusterName,
 					APIVipDNSName: swag.String("some dns name"),
 				},
 			})
 			Expect(reply).To(BeAssignableToTypeOf(common.NewApiError(http.StatusBadRequest, errors.Errorf("error"))))
 		})
 
-		It("cluster update failure on inventory refresh failure", func() {
```

Comment:
Why was this test removed?

Reply:
a rebase error. fixed.

Reply:
I still can't find this test in the recent version you uploaded

Reply:
In V2UpdateCluster there is currently no flow of refreshing hosts, so the entire test is not relevant for v2 (as for now)